src/stan-users-guide/regression.Rmd

```diff
@@ -43,8 +43,8 @@ model {
 }
 ```
 
-There are `N` observations, each with predictor `x[n]` and
-outcome `y[n]`.  The intercept and slope parameters are
+There are `N` observations. For each observation $n \in N$  we have predictor
```

Comment:
[optional]
I've used more British comma style throughout, so I'd prefer a comma after "$n \in N$".

Reply:
Hmm, can I change that here somehow?

Reply:
Fixed